src/controllers/HistoryController.ts

```diff
@@ -200,6 +183,13 @@ export class HistoryController extends RootComponent {
     // Even though it's technically possible that many property changed at the same time since the last state,
     // the backend UA service does not support multiple search cause for a single search event.
     // So we find the first event that match (if any), by order of importance (query expression > sort > facet)
+
+    if (!this.usageAnalytics) {
```

Comment:
👍 

Maybe a small UT that check that passing no usage analytics to this module does not throw any errors when the URL is modified directly.. ?